### PR TITLE
[Improvement]kill yarn type task optimization

### PR DESCRIPTION
**Pull request by gitriver**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/development/pull-request.html before opening a pull request.*

## What is the purpose of the pull request

*(For example: This pull request adds checkstyle plugin.)*

## Brief change log

*(for example:)*
  - *Add maven-checkstyle-plugin to root pom.xml*

## Verify this pull request

*(Please pick either of the following options)*

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

*(example:)*

  - *Added dolphinscheduler-dao tests for end-to-end.*
  - *Added CronUtilsTest to verify the change.*
  - *Manually verified the change by testing locally.*

### CODE REVIEW

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java [hunk 1]
```diff
@@ -129,11 +127,19 @@ public void process(Channel channel, Command command) {
...
+                case FLINK:
+                    // find log and kill yarn job
+                    appIds = killYarnJob(Host.of(taskExecutionContext.getHost()).getIp(),
+                            taskExecutionContext.getLogPath(),
+                            taskExecutionContext.getExecutePath(),
+                            taskExecutionContext.getTenantCode());
+                    break;
+                default:
+                    appIds = Collections.EMPTY_LIST;
+            }
```
Comment (zhuangchong) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java]: hive-sql,mr,YARN tasks started in shell/pythod scripts, and so on. These all involve killing yarn tasks, and this change is not recommended here

**Comment by CalvinKirs** [comment]
code style check fails, you can refer this docs:https://dolphinscheduler.apache.org/en-us/community/development/pull-request.html

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java (hunk 1 above)
Comment (gitriver) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java]: not all job is run on yarn cluster, 
this place need optimize

**Comment by lenboo** [comment]
no file change, close this pr

**Pull request closed without merging**